lib/config.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+function configure () {
+  return Object.freeze({
+    "server": Object.freeze({
```

Comment:
@Zooloo2014 Is there a reason you stored the object in JSON instead of a plain javascript object? If not could you please change it to a javascript object to match the other mediator configs - unless @BMartinos we should be converting those to JSON?

Reply:
This was originally in a .JS. @de-laz  requested this change to json. We agreed previously that the mediator config would stay as JSON and be injected.